__init__.py

```diff
@@ -131,6 +161,23 @@ def _schedule_date_update(self):
             self.update_date, when=date_update_start_time, frequency=ONE_MINUTE
         )
 
+    def _schedule_skill_datetime_update(self):
+        """Check for a weather update every fifteen minutes."""
+        self.schedule_repeating_event(
+            self.update_skill_datetime, when=datetime.now(), frequency=ONE_HOUR
+        )
+
+    def update_skill_datetime(self):
+        """Sets the skill update date for display on the home screen."""
+        skill_update_datetime = ""
+        skill_info_path = Path("~/.mycroft/skills.json").expanduser()
```

Comment:
This will move soon with the XDG changes. Is there a config value that points to this location by any chance?

Reply:
it is hardcoded in MSM, which is where I found it.  don't think a config value exists.